### DISCUSSION TITLE
无法登录

### DISCUSSION THREAD

**Discussion by Shapaper** [DreamVoid/MiraiMC#193] [问答 / Q&A]
![image](https://user-images.githubusercontent.com/88654772/177520152-e5d0ee37-3a61-46a9-830f-e08a7b465380.png)
![image](https://user-images.githubusercontent.com/88654772/177520168-2b0801da-d79c-4644-ac3e-52f10b0dca1f.png)
我的理解是：mirai login QQ号 API密钥[非QQ密码]
然而miraimc似乎将API密钥识别成密码了

复现方法
1. 打开mirai的http-api功能
2. 通过mirai login登录外部mirai

**Comment by DreamVoid** [comment]
那么你有没有尝试过在协议部分填上HTTPAPI

**Reply by Shapaper** [reply]
![image](https://user-images.githubusercontent.com/88654772/177730653-cd74f25c-8a40-46b8-b6c7-dd1c292d73d4.png)
这个嘛？

**Reply by Shapaper** [reply]
![image](https://user-images.githubusercontent.com/88654772/177734459-a9c4f842-b238-45c1-8fbf-c20990a01f23.png)
![image](https://user-images.githubusercontent.com/88654772/177734515-ab693092-23f8-46fd-992b-8b73a0d36e88.png)
huib会报错

**Reply by Shapaper** [reply]
@DreamVoid

**Reply by DreamVoid** [reply]
去开issue

**Reply by Shapaper** [reply]
> 去开题

![image](https://user-images.githubusercontent.com/88654772/178139904-a32389d7-6280-466b-a246-32a4c4ff832d.png)
请问有使用文档吗？
